### ISSUE TITLE
dose not work well with others

### ISSUE THREAD

**Issue by Rob2k9** [StartBootstrap/startbootstrap-simple-sidebar#106]
i am having issues when using this it does not seem to want to work well with other common bootstrap eliments such as

datatables / fontawsome 

if i include the fontawsone js file it dosent show icons if i add in a datatable it dosent seem to show the table properly or update the css to show the table properly

**Comment by drivemetech** [comment]
Yes especially when you are trying to use it with django take a look at this please:
[Stack Overflow question](https://stackoverflow.com/questions/72990541/how-can-i-design-sidebar-using-bootstrap-5/72997150#72997150)
